openedx_lti_tool_plugin/deep_linking/tests/test_urls.py

```diff
@@ -0,0 +1,36 @@
+"""Test urls module."""
+from uuid import uuid4
+
+from django.test import TestCase
+from django.urls import resolve, reverse
+
+from openedx_lti_tool_plugin.deep_linking.views import DeepLinkingFormView, DeepLinkingView
+
+
+class TestDeepLinkingViewUrlPatterns(TestCase):
+    """Test DeepLinkingView Django URL configuration."""
+
+    def test_view_url(self):
+        """Test view URL."""
+        self.assertEqual(
+            resolve(
+                reverse('1.3:deep-linking:root'),
+            ).func.view_class,
+            DeepLinkingView,
+        )
+
+
+class TestDeepLinkingFormViewUrlPatterns(TestCase):
+    """Test DeepLinkingFormView Django URL configuration."""
+
+    def test_view_url(self):
+        """Test view URL."""
+        self.assertEqual(
+            resolve(
+                reverse(
+                    '1.3:deep-linking:form',
+                    kwargs={'launch_id': uuid4()},
+                ),
+            ).func.view_class,
+            DeepLinkingFormView,
+        )
```

Comment:
@alexjmpb Because we can make sure the pattern can be resolved properly and that the kwargs that will be sent to the view are correctly named on the urlpattern

Reply:
But isn't resolving the urls already being tested by Django? Since it's provided by Django I think we can omit the testing.

Reply:
@alexjmpb This is not testing that the URL resolving works but rather that the declaration of the URL pattern is correct, for example I could unit test a view that calls reverse with X pattern and it will always work since it's a mock that's just testing the call, with this we make sure that such pattern is properly declared on urls.py